src/components/ToastCard/_toast-card.scss

```diff
@@ -0,0 +1,29 @@
+@import "vanilla-framework/scss/vanilla";
```

Comment:
Vanilla has toast styles already. Did those not work for some reason?

Reply:
As discussed, Vanilla does not have toast but notification styles which are different.